src/egon/data/airflow/dags/pipeline.py

```diff
@@ -352,30 +355,47 @@
         ]
     )
 
+    prepare_pypsa_eur = PreparePypsaEur(
+        dependencies=[
+            weather_data,
+            data_bundle,
+        ]
+    )
+
+
+    geothermal_potential_germany = GeothermalPotentialGermany(
+        dependencies=[
+            data_bundle,
+            district_heating_areas,
+        ]
+        )
+
+    # Deal with electrical neighbours
+    foreign_lines = ElectricalNeighbours(
+        dependencies=[prepare_pypsa_eur, tyndp_data]
```

Comment:
I think ElectricalNeighbours requires that the task _electrical_neighbours_egon100_ is previously executed. The mentioned task belongs to _run_pypsaeur_ but the dependency points to _ PreparePypsaEur_.

Reply:
There is a dependency the other way around: electrical_neighbors_egon100 depends on ElectricalNeighbours. All AC buses are now created and inserted within ElectricalNeighbours. In electrical_neighbors_egon100, components are attached to these buses, considering the indices inserted by ElectricalNeighbours. 
Could you explain why you see the need for the dependency you mentioned?